src/app-layer-smtp.c

```diff
@@ -263,6 +264,38 @@ static void SMTPConfigure(void) {
     /* Pass mime config data to MimeDec API */
     MimeDecSetConfig(&smtp_config.mime_config);
 
+    ConfNode *t = ConfGetNode("app-layer.protocols.smtp.inspected-tracker");
+    ConfNode *p = NULL;
+
+    if (t == NULL)
+        return;
+
+    TAILQ_FOREACH(p, &t->head, next) {
+        if (strcasecmp("content-limit", p->name) == 0) {
+            if (ParseSizeStringU32(p->val, &content_limit) < 0) {
+                SCLogError(SC_ERR_SIZE_PARSE, "Error parsing content-limit "
+                           "from conf file - %s. Killing engine", p->val);
+                exit(EXIT_FAILURE);
```

Comment:
Can you remove the exits and do proper error handling? Warning+default value is what I have in mind.